apps/frontend/src/app/3_organisms/ERC20BridgeDialog/hooks/useBridgeAssets.ts

```diff
@@ -0,0 +1,39 @@
+import { useMemo } from 'react';
+
+import { ChainIds } from '@sovryn/ethers-provider';
+
+import { useBridgeService } from './useBridgeService';
+
+export const useAssetsBySourceChain = (
+  sourceChainId: typeof ChainIds | undefined,
+) => {
```

Comment:
I believe it should be like this since the `getAssetsBySourceChain` expects a `value` but not a `type`

```suggestion
export const useAssetsBySourceChain = (sourceChainId: ChainIds | undefined) => {
```